result/block-spacing.js

```diff
@@ -0,0 +1,16 @@
+/*eslint block-spacing: "error"*/
+
+function foo() {return true;}
+if (foo) { bar = 0;}
+/*eslint block-spacing: "error"*/
+
+function foo() { return true; }
```

Comment:
__[development][eslint]__
* Error - 'foo' is already defined (no-redeclare)
* Error - Unexpected space(s) after '{'. (block-spacing)
* Error - Unexpected space(s) before '}'. (block-spacing)